src/layman/common/rest.py

```diff
@@ -148,11 +148,25 @@ def get_publications(publication_type, user, request_args=None, workspace=None):
     if consts.ORDER_BY_FULL_TEXT in order_by_list:
         ordering_full_text = full_text_filter
 
+    limit = request_args.get(consts.LIMIT)
+    if limit:
+        try:
+            limit = int(limit)
+        except ValueError as exc:
+            raise LaymanError(2, {'parameter': consts.LIMIT, 'expected': {'text': f'Integer', }}) from exc
```

Comment:
I prefer only non-negative integers to be valid. Regular expression could be used in similar way as bbox.